doc/content/getting-started/installation/configuration/index.md

```diff
@@ -58,11 +58,9 @@ Download the example `ttn-lw-stack-docker.yml` for {{% tts %}} Open Source <a hr
 {{< /tabs/tab >}}
 {{< /tabs/container >}}
 
-These example configuration files contain all of the configuration settings you need to run {{% tts %}} for development. Be sure to update `ttn-lw-stack-docker.yml` with your server address, generate keys in `http.cookie` and set `console.oauth.client-secret`.
+These example configuration files contain all of the configuration settings you need to run {{% tts %}} for development. Be sure to update `ttn-lw-stack-docker.yml` with your server address.
```

Comment:
Why shouldn't they generate them anymore ?

Reply:
they should, but the installation will not fail if they don't, so those can be included in the detailed description of production settings. otherwise, this list needs to be made much longer, it should also include tenant admin keys, noc secret, etc, and i want to keep this part as simple as possible.

Reply:
But if I follow this tutorial, at which point do you tell me how to do the production settings ? We don't have a textual page for that yet, don't we ?